core_perception/lidar_localizer/nodes/ndt_mapping_tku/mapping.cpp

```diff
@@ -14,13 +14,12 @@
 #include "sensor_msgs/PointCloud2.h"
 #include "std_msgs/String.h"
 #include "tf/message_filter.h"
-#include "velodyne_pointcloud/point_types.h"
+#include "velodyne_pcl/point_types.h"
 #include "velodyne_pointcloud/rawdata.h"
 
 #include <message_filters/subscriber.h>
 #include <message_filters/time_synchronizer.h>
 
-pcl::PointCloud<velodyne_pointcloud::PointXYZIR> map;
```

Comment:
Yes, it was. I probably should wait on clean-up items like this for a later pass. Since the port to ROS 2 will involve reworking, that would be a better time to make code clean-up-related changes. I'll revert.